src/wp-includes/fonts/class-wp-font-utils.php

```diff
@@ -29,12 +29,15 @@ class WP_Font_Utils {
 	 * @param string $item A font family name.
 	 * @return string The font family name with surrounding quotes, if necessary.
 	 */
-	private static function maybe_add_quotes( $item ) {
+	public static function normalize_quoted_font_family_name( $item ) {
 		// Matches strings that are not exclusively alphabetic characters or hyphens, and do not exactly follow the pattern generic(alphabetic characters or hyphens).
 		$regex = '/^(?!generic\([a-zA-Z\-]+\)$)(?!^[a-zA-Z\-]+$).+/';
 		$item  = trim( $item );
 		if ( preg_match( $regex, $item ) ) {
 			$item = trim( $item, "\"'" );
+			$item = str_replace( [ "\r\n", "\r", "\n" ], '\\A', $item );
```

Comment:
Good catch on [normalizing the newlines](https://www.w3.org/TR/css-syntax-3/#input-preprocessing) and replacing them.

However the replacement may be problematic because it could be followed by hex digits which would then be interpreted as part of the escape sequence. The safest thing to do here is to include a trailing space character to terminate the escape sequence:

```suggestion
			$item = str_replace( [ "\r\n", "\r", "\n" ], '\\A ', $item );
```